distributions/raspijamulus.sh

```diff
@@ -62,10 +65,10 @@ if [ -d "jack2" ]; then
   echo "The Jack2 directory is present, we assume it is compiled and ready to use. If not, delete the jack2 directory and call this script again."
 else
   git clone https://github.com/jackaudio/jack2.git
-  cd jack2
+  cd jack2 || exit 1
   git checkout v1.9.20
-  ./waf configure --alsa --prefix=/usr/local --libdir=$(pwd)/build
-  ./waf -j${NCORES}
+  ./waf configure --alsa --prefix=/usr/local --libdir="$(pwd)"/build
```

Comment:
`"--libdir=$(pwd)/build"` is nicer (or `--libdir="$(pwd)/build"` - but splitting the /build off doesn't look nice.)